README.md

```diff
@@ -20,7 +20,7 @@
 
 <hr>
 
-Prisma Client Go is an **auto-generated query builder** that enables **type-safe** database access and **reduces boilerplate**. You can use it as an alternative to traditional ORMs such as GOORM, sqlboiler and most database-specific tools.
+Prisma Client Go is an **auto-generated query builder** that enables **type-safe** database access and **reduces boilerplate**. You can use it as an alternative to traditional ORMs such as gorm, xorm, sqlboiler and most database-specific tools.
```

Comment:
This actually doesn't just fix a typo, it also names a new one ;)

Reply:
oh. yes i'd added one additional ORM name also.